app/controllers/staff/activity_forms_controller.rb

```diff
@@ -46,6 +47,10 @@ def update
     update_activity_attributes_except_dates
     record_auditable_activity
 
+    if @activity.wizard_complete?
+      reset_sector_answer if step == :sector_category
```

Comment:
I can see how we can make an easy extension to this for more steps, like geography 👍